physics/body_centered_non_rotating_dynamic_frame_body.hpp

```diff
@@ -40,10 +40,10 @@ BodyCentredNonRotatingDynamicFrame<InertialFrame, ThisFrame>::ToThisFrameAtTime(
 template<typename InertialFrame, typename ThisFrame>
 void BodyCentredNonRotatingDynamicFrame<InertialFrame, ThisFrame>::
 WriteToMessage(not_null<serialization::DynamicFrame*> const message) const {
-  message->MutableExtension(
-      serialization::BodyCentredNonRotatingDynamicFrame::
-          body_centred_non_rotating_dynamic_frame)->set_centre(
-              ephemeris_->serialization_index_for_body(centre_));
+  message
+      ->MutableExtension(
+          serialization::BodyCentredNonRotatingDynamicFrame::extension)
+      ->set_centre(ephemeris_->serialization_index_for_body(centre_));
```

Comment:
We put the `->` at the end of the line, contrary to clang-format (I'm not sure where we put the `.`).